<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Packets</class>
 <widget class="QWidget" name="Packets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxUsBetweenBytes">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="suffix">
    <string>, us</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBetweenBytes">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Задержка</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
